### ISSUE TITLE
include(nested-router) + HyperlinkedIdentityField

### ISSUE THREAD

**Issue by tlpriest** [alanjds/drf-nested-routers#76]
I'm getting improper hyperlinks with the following configuration

```
# myapp/urls.py

urlpatterns = [
    url(r'^$', RedirectView.as_view(url=reverse_lazy('v1:swagger:django.swagger.base.view'), permanent=False)),
    url(r'^user/', include('myapp.account.urls_v1_0', namespace='user')),
...
    url(r'^sponsor/', include('myapp.sponsor.urls_v1_0', namespace='sponsor')),
...
]
```

```
# myapp/sponsor/urls.py
router = routers.DefaultRouter()
router.register(r'sponsor', SponsorViewSet, base_name='sponsor')

member_router = routers.NestedSimpleRouter(router, r'sponsor', lookup='sponsor')
member_router.register(members', MemberViewSet, base_name='members')
...
```

```
# serializers.py
...
members = HyperlinkedIdentityField(view_name='v1.0:sponsor:members-list', lookup_url_kwarg='sponsor_pk')
...
```

In this configuration, the returned hyperlink is:

```
u'http://localhost/v1.0/sponsor/sponsor/724/followers/'
```

Changing to:

```
# myapp/sponsor/urls.py
router = routers.DefaultRouter()
router.register(r'', SponsorViewSet, base_name='sponsor')

member_router = routers.NestedSimpleRouter(router, r'', lookup='sponsor')
member_router.register(members', MemberViewSet, base_name='members')
...
```

yields: 

```
u'http://localhost/v1.0/sponsor//724/followers/'
```

That "works," but... So, changing to:

```
# myapp/sponsor/urls.py
router = routers.DefaultRouter()
router.register(None, SponsorViewSet, base_name='sponsor')

member_router = routers.NestedSimpleRouter(router, None, lookup='sponsor')
member_router.register(members', MemberViewSet, base_name='members')
...
```

yields:

```
u'http://localhost/v1.0/sponsor/None/724/followers/'
```

BaseRouter is simply shoving that prefix value onto a list. My guess is that something in NestedSimpleRouter.**init**() needs to check for '' or None but I'm not sure where. Or is this simply user error?

**Comment by tlpriest** [comment]
This also seems to affect paginators. I tried setting the myapps/urls.py namespace to '' but no joy.

**Comment by alanjds** [comment]
Just released v0.90.0 with #91. It could had fixed this. Can you please try again and inform if yet broken?
